### PR TITLE
feat(limit-conn): support multiple variables as key

### PR DESCRIPTION
**Pull request by Xunzhuo**
### What this PR does / why we need it:

 support multiple variables as key in plugin limit-conn.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: t/plugin/limit-conn2.t [hunk 1]
```diff
@@ -308,3 +308,134 @@ request latency is nil
...
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                        "plugins": {
+                            "limit-conn": {
+                                "conn": 2,
+                                "burst": 1,
+                                "default_conn_delay": 0.1,
+                                "rejected_code": 503,
+                                "key": "$http_a $http_b",
```
Comment (tzssangglass) [inline t/plugin/limit-conn2.t:328]: Are the two variables separated by a space? Depending on the document it is separated by `|`.

**Review by spacewander — APPROVED** [review]

File: t/plugin/limit-conn2.t (hunk 1 above)
Comment (Xunzhuo) [reply t/plugin/limit-conn2.t:328]: Thanks for your tips, two variables are separated by a space, doc updated.

**Comment by Xunzhuo** [comment]
@spacewander Coud you take a look at the CI failure?

File: t/plugin/limit-conn2.t (hunk 1 above)
Comment (Xunzhuo) [reply t/plugin/limit-conn2.t:328]: @tzssangglass  would you like to take a look at this PR again? doc updated.

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**